pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>com.google</groupId>
   <artifactId>healthcare-api-dicom-fuse</artifactId>
-  <version>1.0.1</version>
+  <version>1.1.0</version>
```

Comment:
Let's use 1.0.2 and save 1.1 for major new feature launches.

Reply:
Ok, I'll change.